util/task.py

```diff
@@ -232,6 +232,7 @@ def write_base_values(dt, session):
 
         session.commit()
     except Exception as e:
+        logger.error("Error while calculating pkv for %s at %s", user.meter_id, dt)
```

Comment:
As discussed before, one cannot access the parameter `user.meter_id` at this place in the code. Please move the log message to another place where the variable can be accessed or provide a different exception handling altogether.